toolkit/offchain/src/csl.rs

```diff
@@ -75,6 +75,7 @@ pub(crate) fn get_builder_config(
 ) -> Result<TransactionBuilderConfig, JsError> {
 	let protocol_parameters = &context.protocol_parameters;
 	TransactionBuilderConfigBuilder::new()
+		.prefer_pure_change(true)
```

Comment:
This is the reason for which tests are failing. I don't know if it is required. I've reverted this change from my own work.